corgi/tasks/brew.py

```diff
@@ -300,3 +307,24 @@ def save_module(softwarebuild, build_data) -> ComponentNode:
     # TODO: recurse components from build_data["meta"]["components"]
 
     return node
+
+
+def load_brew_tags() -> None:
+    for ps in ProductStream.objects.all():
+        brew = Brew()
+        for brew_tag, inherit in ps.brew_tags.items():
+            try:
+                builds = brew.get_builds_with_tag(brew_tag, inherit)
+            except GenericError as exc:
+                logger.error("Got error getting builds for %s: %s", brew_tag, exc)
```

Comment:
Is it intentional to keep running a task and just log the error ? Alternately celery would trap this right ?

Reply:
Ok, removed the try/catch from here, and letting Celery retry this in case of error.